### ISSUE TITLE
Style/SymbolArray ignores arrays of 1 element, even when MinSize is set to 0 (With root cause)

### ISSUE THREAD

**Issue by TikiTDO** [bbatsov/rubocop#5897]
## Expected behavior

When using:

```
Style/SymbolArray:
  Enabled: true
  MinSize: 1
```

I expect `[:symbol]` to be auto-corrected to `%i[symbol]`. 

## Actual behavior

Rubocop ignores `[:symbol]`. 

The issue is caused by [this line](https://github.com/bbatsov/rubocop/blob/3988d2ffb28e57f1f5b6075762d0feee422bb513/lib/rubocop/cop/mixin/array_syntax.rb#L11), which does a check of `node.values.size > 1` when it should be doing a check of `node.values.size >= 1`.

## Steps to reproduce the problem

1. Create a `file.rb` with an array of one symbols `[:symbol]`
2. Enable `Style/SymbolArray` with a `MinSize` of 1 or 0. 
3. Run `rubocop --auto-correct file.rb`

The array is not detected as an issue, and is not corrected.

## RuboCop version

```
└> rubocop -V
0.55.0 (using Parser 2.5.1.0, running on ruby 2.5.1 x86_64-linux)
```
